model/commitqueue/github_pr_message.go

```diff
@@ -18,16 +18,17 @@ const (
 )
 
 type GithubMergePR struct {
-	PatchSucceeded bool   `bson:"patch_succeeded"`
-	URL            string `bson:"url"`
-	ProjectID      string `bson:"project_id"`
-	Owner          string `bson:"owner"`
-	Repo           string `bson:"repo"`
-	Ref            string `bson:"ref"`
-	PRNum          int    `bson:"pr_number"`
-	CommitMessage  string `bson:"commit_message"`
-	CommitTitle    string `bson:"commit_title"`
-	MergeMethod    string `bson:"merge_method"`
+	PastTenseStatus string `bson:"past_tense_status"`
```

Comment:
does changing this field to a bool break anything existing? 

The naming makes it hard to understand. Souldn't this just be Status or Outcome?

Reply:
I think I've cleaned up all the places that were expecting a bool.
Now renamed PastTenseStatus to Status.